kibbeh/src/shared-hooks/useSetDeaf.ts

```diff
@@ -2,14 +2,14 @@ import { Wrapper } from "@dogehouse/kebab";
 import { useDeafStore } from "../global-stores/useDeafStore";
 import { useWrappedConn } from "./useConn";
 
+export const setDeaf = (conn: Wrapper, value: boolean) => {
```

Comment:
This is entirely unrelated, but was causing the `lint` step to fail in the GH actions.

Reply:
pull latest and push, should be fixed